gramjs/client/uploads.ts

```diff
@@ -126,7 +124,7 @@ export async function uploadFile(
 
     const partSize = getAppropriatedPartSize(bigInt(size)) * KB_TO_BYTES;
     const partCount = Math.floor((size + partSize - 1) / partSize);
-    const buffer = await getFileBuffer(file, size, fileParams.maxBufferSize);
+    const buffer = await getFileBuffer(file, size, fileParams.maxBufferSize || /* 2G */ 2 ** 31 - 1);
```

Comment:
can we maybe have this 2GB in a separate variable (constant) on top of this file. 
The rest is ok.

Reply:
I think /* 2G */ mark is enough for readability, but the constant method is also a good way.

Reply:
it's just so that we can change it easily later if needed.